install/installation/installation.go

```diff
@@ -74,6 +74,8 @@ type Installation struct {
 	TillerYaml string
 	// InstallerYaml is a content of yaml file with all resources related to and required by Installer
 	InstallerYaml string
+	// InstallerCRYaml is a content of yaml file with the list of the packages being installed
+	InstallerCRYaml string
```

Comment:
Currently, Installation CR is a part of InstallerYaml and this API should be aligned with it. If we want to make such a change, then the Installation CR should be moved out of InstallrYaml artifact.

Reply:
By default on master the installation CR is not a part of the InstallerYaml. Currently during the release process we are merging the two. This PR: https://github.com/kyma-project/test-infra/pull/2517 will change the behavior so that we get two files on release page similar to the one in master.